src/api/java/com/minecolonies/api/client/render/modeltype/BipedModelType.java

```diff
@@ -1,78 +1,72 @@
 package com.minecolonies.api.client.render.modeltype;
```

Comment:
Who put this in the API? Did I? Because this is an internal implementation detail, and should never be here in the first palce.